### ISSUE TITLE
About errors upon decrypting files

### ISSUE THREAD

**Issue by 11-59pm** [Francesco149/klbvfs#3]
First of all, thank you for your excellent work!
Using your script, I managed to decrypt my db files, but when I try to decrypt my files using -dump argument, it continuously spit out this error:

![aesrfhha](https://user-images.githubusercontent.com/50797622/92499747-33899880-f237-11ea-9c02-756591394a22.PNG)

Could you help me how to use the script properly..? Any help would be appreciated.
Best regards.

**Comment by Francesco149** [comment]
it means that it can't find `asset_a_ja_0.db.*` or  `asset_a_ko.db.*`

maybe it has an unusual name. if so, just add it/change it in the code

**Comment by garamika** [comment]
Thanks for your help!
Now I've edited the name and it does not display the same error.
But I've encountered another one.

![1111](https://user-images.githubusercontent.com/50797622/92590433-30de8000-f2d7-11ea-9fcc-de58ec058fdc.PNG)
![2222](https://user-images.githubusercontent.com/50797622/92590440-35a33400-f2d7-11ea-866e-3eecd95897cf.PNG)

Though the db seems to open w/ sqlite3 but it constantly displays the same error.
